setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+
+with open("README.md", 'r') as f:
+    long_description = f.read()
+
+setup(name='review-rot',
```

Comment:
Yes, the file was included in the former PR but this adds 'PyGtihub' as 'install_requires'.